packages/utilities/fast-design/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "fast-design",
+  "version": "0.0.1",
+  "description": "A collection of design utilities supporting basic styling and Adaptive UI",
+  "type": "module",
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "tsc -p ./tsconfig.build.json && yarn doc",
+    "build:tests": "tsc -p ./tsconfig.test.json",
+    "clean:dist": "node ../../../build/clean.js dist",
+    "doc": "api-extractor run --local",
+    "doc:ci": "api-extractor run",
+    "prepare": "yarn clean:dist && yarn build",
+    "prettier": "prettier --config ../../../.prettierrc --write \"**/*.ts\"",
+    "prettier:diff": "prettier --config ../../../.prettierrc \"**/*.ts\" --list-different",
+    "test": "yarn build:tests && yarn eslint && yarn unit-tests && yarn doc",
+    "eslint": "eslint . --ext .ts",
+    "eslint:fix": "eslint . --ext .ts --fix",
+    "unit-tests": "mocha",
+    "unit-tests:watch": "mocha --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/microsoft/fast.git"
+  },
+  "author": {
+    "name": "Microsoft",
+    "url": "https://discord.gg/FcSNfg4"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/microsoft/fast/issues/new/choose"
+  },
+  "homepage": "https://fast.design",
+  "dependencies": {
+    "@microsoft/fast-colors": "^5.2.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.2.11",
+    "@types/mocha": "^8.2.0",
+    "chai": "^4.2.0",
+    "mocha": "^8.2.1",
+    "@microsoft/eslint-config-fast-dna": "^2.0.0",
+    "eslint-config-prettier": "^6.10.1",
+    "prettier": "2.0.2",
+    "typescript": "^3.9.0"
+  }
+}
```

Comment:
Should there be code coverage for this repo? If so, check out the workflow validation file to include the path where data is dumped.